html/elements/script.json

```diff
@@ -548,6 +548,224 @@
                 "deprecated": false
               }
             }
+          },
+          "speculationrules": {
+            "__compat": {
+              "spec_url": "https://wicg.github.io/nav-speculation/speculation-rules.html#speculation-rules-script",
+              "support": {
+                "chrome": {
+                  "version_added": "105",
+                  "notes": "Initial support included same-origin prerendering only."
```

Comment:
I think you need version ranges inline here and the sub feature "crossorigin_prerending" should be removed.

105 : partial support because same-origin only
109: full support

Reply:
Good call; done.